specification/compliance.md

```diff
@@ -10,14 +10,24 @@ Cloud Service Providers (CSPs) and some SaaS providers generate detailed cost an
 
 Most other providers typically offer billing datasets that lack the same level of detail. However, those less-detailed datasets meeting an essential set of requirements can enable some important FinOps capabilities like invoice reconciliation, data ingestion, and data normalization. Simplified versions of additional FinOps capabilities like cost reporting, chargeback, showback, and budgeting/forecasting can also be performed using less-detailed billing datasets when these essential requirements are met.
 
-The essential set of data requirements needed to perform the most important FinOps capabilities is referred to as the 'FOCUS Essential' requirements.
-
 ## Meeting FOCUS Requirements
 
 *This section is normative.*
 
-All [columns](#columns) presented in a FOCUS-compliant dataset MUST meet the requirements specified in the normative prose of each column. These requirements may include nullability, value and format restrictions, and other scenario-specific requirements. Additionally, all FOCUS-compliant datasets MUST meet all [attributes](#attributes) defined in the FOCUS specification.
+FOCUS defines three levels of compliance: Basic, Standard, Full, and Enhanced. All [columns](#columns) presented in a FOCUS-compliant dataset MUST meet the requirements specified in the normative prose of each column. Compliance (and compliance level) MUST be applied to the full dataset only and cannot be achieved if part of a dataset is excluded. These requirements may include nullability, value and format restrictions, and other scenario-specific requirements. Additionally, all FOCUS-compliant datasets MUST meet all [attributes](#attributes) defined in the FOCUS specification.
 
-The set of columns defined as FOCUS Essential MUST be present in a FOCUS-compliant dataset. The non-FOCUS Essential columns MAY be omitted from the billing dataset based on the conditions defined in the normative prose of such columns.
+Each column defines the compliance level at which that column is required. These columns MUST be present in the dataset to conform to the specified compliance level of FOCUS. Additional columns MAY be omitted from the dataset based on the conditions defined in the normative prose of such columns.
 
 For example, if a provider does not support commitment-based discounts, the columns related to commitment-based discounts can be omitted. Providers that support the commitment-based discount construct must provide the commitment-based discount columns and meet the requirements defined in the normative prose of those columns. If a provider does not support commitment-based discounts and chooses to include commitment-based discount-related columns, the data presented in those columns must still meet all requirements specified in the normative prose of the commitment-based discount columns that were included.
+
+### Compliance Levels
```

Comment:
Suggestion: Consider adding a new document section with a table that connects `columns` to their respective `compliance levels`. This will aid users in identifying the elements of each compliance level. Furthermore, having a `Compliance Level` entry in every column description could be redundant with the inclusion of this compliance section.

Reply:
Fwiw, I do see value in both. That way people can see what level the column is at a glance when reviewing a single column without having to bounce back and forth in the spec. We talked about having a similar summary section a month or so back. I'm all for that!